protocol/contracts/fakes/FakeTempleLineofCredit.sol

```diff
@@ -0,0 +1,29 @@
+pragma solidity ^0.8.17;
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Oud TODO)
+
+import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+import '@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol';
```

Comment:
It's a fake only, so no biggie - but this should also be explicit on the contract it pulls in
`import {SafeERC20} from '@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol';`

Reply:
fixed in [6dcd2d9](https://github.com/TempleDAO/temple/pull/760/commits/6dcd2d9703508819451179aa0369ffc01d18a63a)